test/test_dremio_arctic.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+import pytest
+import sqlalchemy
+
+from . import conftest
+
+LOGGER = logging.getLogger(__name__)
+
+FULL_ROWS = """[(1, 81297389127389213, Decimal('9112.229'), 9192.921875, 9292.17272, 'ZZZZZZZZZZZZ', b'AAAAAAAAA', datetime.datetime(2020, 4, 5, 15, 8, 39, 574000), datetime.date(2020, 4, 5)
+, datetime.time(12, 19, 1), True), (2, 812123489127389213, Decimal('6782.229'), 2234193.0, 9122922.17272, 'BBBBBBBBB', b'CCCCCCCCCCCC', datetime.datetime(2020, 4, 5, 15, 8, 39, 574000), datetime.date(2022,
+4, 5), datetime.time(10, 19, 1), False)]"""
+
+
+@pytest.fixture(scope='session')
+def engine():
```

Comment:
Same as in test_dremio.py, is this fixture needed? I only see it being used in test_row_count(engine), but the engine used comes from conftest.get_cloud_engine().